src/styles/breadcrumb.scss

```
@@ -0,0 +1,28 @@
+$breadcrumb-item-color: $color-tomato;
+$breadcrumb-item-active-color: $color-dark-slate-gray;
+$breadcrumb-item-hover-color: $color-tomato;
+$breadcrumb-item-padding-horizontal: $space-small;
+$breadcrumb-item-separator-color: $color-gray;
+
+
+@import "bulma/sass/components/breadcrumb.sass";
+
+
+nav.breadcrumb > ul {
+
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 19px;
```

Use line heights as fraction of font-size so 19/13 = 1.4615384615.
```suggestion
  line-height: 1.4615384615;
```

@panchovm this is not the same as the other line heights which all use a constant ratio. Is that intentional?

---

If that helps -
font-family: Source Sans Pro;
font-style: normal;
font-weight: 600;
font-size: 13px;
line-height: 19px;
This is in the design library.